From b655018c04d218a6fe185709be0cb0a4158a9477 Mon Sep 17 00:00:00 2001
From: skyel <razvan.ghitulete@gmail.com>
Date: Wed, 9 Apr 2014 18:10:03 +0200
Subject: [PATCH 15/18] edfi: allocating space for __force_order by declaring
 static

We allocate space for said variable even though we might have multiple
instances across the binary because otherwise clang and llvm refuse to link
due to the undefined, although declared, symbol

Signed-off-by: skyel <razvan.ghitulete@gmail.com>
---
 arch/x86/include/asm/special_insns.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/x86/include/asm/special_insns.h b/arch/x86/include/asm/special_insns.h
index 645cad2..7033cad 100644
--- a/arch/x86/include/asm/special_insns.h
+++ b/arch/x86/include/asm/special_insns.h
@@ -16,7 +16,7 @@ static inline void native_clts(void)
  * all loads stores around it, which can hurt performance. Solution is to
  * use a variable and mimic reads and writes to it to enforce serialization
  */
-extern unsigned long __force_order;
+static unsigned long __force_order;
 
 static inline unsigned long native_read_cr0(void)
 {
-- 
1.8.3.1

